llama_stack/apis/agents/agents.py

```diff
@@ -225,8 +225,18 @@ def model_post_init(self, __context):
 
 @json_schema_type
 class AgentConfig(AgentConfigCommon):
+    """Configuration for an agent.
+
+    :param model: The model identifier to use for the agent
+    :param instructions: The system instructions for the agent
+    :param name: Optional name for the agent, used in telemetry and identification
+    :param enable_session_persistence: Whether to persist session data
```

Comment:
Should we mark this optional like the others? Not sure it’s essential to call out optional fields explicitly.